sample_workloads/nccltest/gke/templates/nccl_benchmarks.yaml

```diff
@@ -0,0 +1,178 @@
+# yamllint disable
+{{- $requiredVar := .Values.cluster.nNodes | required ".Values.cluster.nNodes is required, add --set cluster.nNodes=2" -}}
+
+{{ $timestamp := now | date "2006-01-02-150405" }}
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: "nccl-benchmarks-{{ $.Release.Name }}-{{ $timestamp }}"
+spec:
+  selector:
+    name: "nccl-benchmarks-{{ $.Release.Name }}-{{ $timestamp }}"
+  clusterIP: None
+---
+{{ $node_count := .Values.cluster.nNodes | int }}
+{{ $superblock_count := .Values.cluster.nSuperblocks | int }}
```

Comment:
Concern about the SB portions (lines 16-17, 35-40). This is dependent on the Kubernetes cluster having a `superblock` label indicating the index of the SB, which is baked into our internal GKE cluster creation process.

Reply:
Correct, I will highlight this in the readme. We have several different flows right now for cluster creation, that each use a different label or mechanism for superblock assignment.